static/truewiki/truewiki.css

```diff
@@ -107,11 +115,18 @@ main {
 
 #pagename {
     color: white;
-    font-size: 300%;
+    font-size: 200%;
     max-width: 700px;
     padding: 5px 0 0 10px;
 }
 
+@media only screen
+    and (min-width: 992px) {
```

Comment:
can you put `and ..` on the same line? In CSS it is nearly impossible to spot otherwise when belongs to the selector and what not :) Similar in the entries lower.